site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7275,8 +7275,20 @@ public function getCurrentQueue() {
         $this->course_db->query($query, [$this->core->getDateTimeNow()->format('Y-m-d')]);
         return $this->course_db->rows();
     }
-
     public function getPastQueue() {
+        $off  = "ORDER BY ROW_NUMBER";
+        $up   = "ORDER BY helper.user_id";
+        $down = "ORDER BY helper.user_id DESC";
+        $orderType = "";
+        if ($_COOKIE['sortStateTimeEntered'] == "up") {
+            $orderType = $up;
+        }
+        elseif ($_COOKIE['sortStateTimeEntered'] == "down") {
+            $orderType = $down;
+        }
```

Comment:
Is this something which should be saved in the browser via cookies and determined on the server side like is being done here, or something we should do on the client side instead?